eduvpn/settings.py

```diff
@@ -29,11 +29,11 @@
     "RWRtBSX1alxyGX+Xn3LuZnWUT0w//B6EmTJvgaAxBMYzlQeI+jdrO6KF",
     "RWQKqtqvd0R7rUDp0rWzbtYPA3towPWcLDCl7eY9pBMMI/ohCmrS0WiM"
 ]
-EDUVPN_ICON = prefix + "/share/icons/hicolor/128x128/apps/eduvpn-client.png"
+EDUVPN_ICON = prefix + "/share/icons/hicolor/128x128/apps/org.eduvpn.client.png"
```

Comment:
A few lines up it says `org.eduvpn.app.linux`. I think there should be a single setting for this that gets interpolated everywhere.

Reply:
is CLIENT_ID the same as application ID?

Reply:
`CLIENT_ID` is used for OAuth, but I'm not sure how that's used by servers.

Reply:
i guess it is unrelated.